### PR TITLE
[RFR] adding separate test for GCE preemtible provision feature

### PR DESCRIPTION
**Pull request by lkhomenk**
Purpose or Intent
=================

__Adding tests__ for feature GCE preemtible VM. It was implemented in product version 5.7 and was agreed to have it in separate test for GCE.

{{pytest: cfme/tests/cloud/test_provisioning.py --use-provider gce_central --long-running}}

### CODE REVIEW

**Comment by dajoRH** [comment]
Lint report for commit 2ea872bd483d5b6299365afe3dd0f20e7f5103bd:

Lint OK! :cake: :punch: :cookie: .


**No commit flaws detected**.
<mergeable/>

*CFME QE Bot*

File: cfme/tests/cloud/test_provisioning.py [hunk 1]
```diff
@@ -95,9 +95,25 @@ def test_provision_from_template(request, setup_provider, provider, testing_inst
              num_sec=1000,
              delay=60,
              handle_exception=True)
-    if provider.type == 'gce' and current_version() >= "5.7":
-        soft_assert('Yes' in instance.get_detail(
-            properties=("Properties", "Preemptible")), "GCE Instance isn't Preemptible")
+    soft_assert(instance.does_vm_exist_on_provider(), "Instance wasn't provisioned")
+
+
+@pytest.mark.uncollectif(lambda provider: provider.type != 'gce')
```
Comment (psav) [inline cfme/tests/cloud/test_provisioning.py]: Just need an uncollect here for 5.6 and we're golden

File: cfme/tests/cloud/test_provisioning.py [hunk 2]
```diff
@@ -95,9 +95,25 @@ def test_provision_from_template(request, setup_provider, provider, testing_inst
...
+def test_gce_preemtible_provision(request, setup_provider, provider, testing_instance, soft_assert):
+    instance, inst_args = testing_instance
+    instance.create(**inst_args)
+    instance.wait_to_appear(timeout=800)
+    provider.refresh_provider_relationships()
```
Comment (psav) [inline cfme/tests/cloud/test_provisioning.py]: @mshriver @mfalesni @izapolsk @jkrocil @RonnyPfannschmidt Here and indeed in the instance create, we should probably have it check/wait for existence by default and have an option not to, having all these different named wait functions/method is annoying.

Comment (psav) [inline cfme/tests/cloud/test_provisioning.py]: @lkhomenk This is not a blocker on this PR thugh, just flagging it for future work

**Comment by lkhomenk** [comment]
added uncollect for 5.6

**Comment by dajoRH** [comment]
Lint report for commit a2f2f1f81d6e07f0f1d6468a8f5855c35d1582ff:

Lint OK! :cake: :punch: :cookie: .


**No commit flaws detected**.
<mergeable/>

*CFME QE Bot*

**Pull request merged**